src/modalities/evaluator.py

```diff
@@ -45,11 +45,11 @@ def evaluate(
     ) -> Dict[str, EvaluationResultBatch]:
         result_dict: Dict[str, EvaluationResultBatch] = {}
         model.eval()
+
+        device = torch.device(self.local_rank) if torch.cuda.is_available() else "cpu"
```

Comment:
This line also appears in `src/modalities/trainer.py` (line 69), we might want to use a function to avoid duplicate code.

Reply:
Fix code duplication later, as we usually run on GPU and only tests need CPU